src/components/ApplicationJSon/CallToAction.vue

```diff
@@ -0,0 +1,368 @@
+  },
+
+  methods: {
+    init: function() {
+      this.headerText = this.document.interactive.header ? this.document.interactive.header.text : ''
+      this.bodyText = this.document.interactive.body ? this.document.interactive.body.text : ''
+      this.footerText = this.document.interactive.footer ? this.document.interactive.footer.text : ''
+      this.url = this.document.interactive.action.parameters.url
+      this.displayText = this.document.interactive.action.parameters.display_text
+    },
+    callToActionSelectedOption: function() {
+      let win = window.open(this.url, '_blank')
+      win.focus()
+    },
+    callToActionSave: function($event) {
+      if (this.errors.any() || ($event && $event.shiftKey)) {
+        return
+      }
+      this.$validator.validateAll().then((result) => {
+        if (!result) return
+
+        this.save({
+          ...this.document.interactive.header,
+          text: this.headerText
+        })
+
+        this.save({
+          ...this.document.interactive.body,
+          text: this.bodyText
+        })
+
+        this.save({
+          ...this.document.interactive.footer,
+          text: this.footerText
+        })
+
+        this.save({
+          ...this.document.interactive.action,
+          button: this.buttonText
+        })
+      })
+    },
+    callToActionEditCancel: function() {
+      this.cancel()
+    }
+  }
+}
+</script>
+
+<style scoped lang="scss">
+@import '../../styles/variables.scss';
+
+.call-to-action .bubble {
+  padding: $bubble-padding;
+  padding-left: 0px;
+  padding-right: 0px;
+  min-width: 206px;
+  text-align: left;
+}
+
+.header-text {
+  padding-top: 0px !important;
+  padding-bottom: 0px !important;
+  padding: $bubble-padding;
+}
+
+.body-text, .footer-text {
+  padding-top: 0px !important;
+  padding-bottom: 10px !important;
+  padding: $bubble-padding;
+}
+
+.button-text {
+  padding-top: 0px !important;
+  padding-bottom: 0px !important;
+  padding: $bubble-padding;
+  border-top: 0.5px solid $color-content-ghost;
+
+  div:first-child {
+    text-align: center;
+    display: inline-flex;
+  }
+}
+
+.menu-list-options-button-icon .lmenu-list-options-button-text {
+  float: left;
+}
+
+.menu-list-options-button-icon {
+  margin-top: 12px;
+}
+.bubble {
+  &.right {
+    .disable-selection {
+      .button-text {
+        .menu-list-options-button-icon {
+          color: $color-surface-1;
+        }
+      }
+    }
+  }
+  
+  &.left {
+    .disable-selection {
+      .button-text {
+        .menu-list-options-button-icon {
+          color: $color-content-default;
+        }
+      }
+    }
+  }
+}
+.blip-card .form-group .help {
+  padding: 0px;
+}
+
+.fixed-options ul {
+    margin: 0px;
+}
+
+.editing .fixed-options ul {
+  margin: 0px -10px;
+}
+
+.fixed-options ul .align-center {
+  display: flex;
+  align-items: center;
+  margin: 0 auto;
+  > bds-icon {
+    margin-right: 10px;
+    color: var(--color-primary, #1E6BF1) !important;    
```

Comment:
```suggestion
    color: $color-primary !important;    
```
Why do not use the color from System design?